src/sentry/deletions/defaults/organizationmember.py

```diff
@@ -0,0 +1,15 @@
+from sentry.models.groupsearchview import GroupSearchView
+
+from ..base import ModelDeletionTask, ModelRelation
+
+
+class OrganizationMemberDeletionTask(ModelDeletionTask):
+    def get_child_relations(self, instance):
+        relations = [
+            ModelRelation(
+                GroupSearchView,
+                {"user_id": instance.user_id, "organization": instance.organization},
```

Comment:
```suggestion
                {"user_id": instance.user_id, "organization_id": instance.organization_id},
```
You should be able to skip loading the Organization here since only the ID is needed